File: mediator/mediator.js
class ChatRoom {
    mostrar(usuario, mensaje) {
        console.log(`[${usuario}]: ${mensaje}`);
    }
}

class Usuario {
    constructor(nombre, chat) {
        this.nombre = nombre;
        this.chat = chat;
    }

    enviar(mensaje) {
        this.chat.mostrar(this.nombre, mensaje);
    }
}

File: mediator/mediator.py
class ChatRoom:
    def mostrar(self, usuario, mensaje):
        print(f"[{usuario}]: {mensaje}")


class Usuario:
    def __init__(self, nombre, chat):
        self.nombre = nombre
        self.chat = chat

    def enviar(self, mensaje):
        self.chat.mostrar(self.nombre, mensaje)
